extension/goplugin/integration_tests/end_to_end_test.go

```diff
@@ -139,10 +139,17 @@ var _ = Describe("Environment", func() {
 				"id":            "testId",
```

Comment:
Context should change to "Resource creation" to "Resource update" (line 129).

Reply:
it tries to create resource three times two of them fails due to validation and third succeeds with status code `http.StatusCreated`. There isn't any update in this test